lib/Util/Util.php

```diff
@@ -206,6 +206,38 @@ public static function secureCompare($a, $b)
         }
     }
 
+    /**
+     * Recursively goes through an array of parameters. If a parameter is an instance of
+     * ApiResource, then it is replaced by the resource's ID.
+     * Also clears out null values.
+     *
+     * @param mixed $h
+     * @return mixed
+     */
+    public static function objectsToIds($h)
+    {
+        if ($h instanceof \Stripe\ApiResource) {
+            return $h->id;
```

Comment:
I wonder if there will be some cases where some resources will not have an id and break this logic and we will regret this.

Reply:
Yeah, I considered that. This would only happen if a user passed a resource without an ID inside a parameter hash though. We could check if the resource has an `id` property and output an error message if not, but I don't think it's worth the hassle.

(stripe-ruby has the same issue fwiw)